src/libs/sensors-logging.ts

```diff
@@ -230,29 +227,32 @@ class DataLogger {
     description: 'Local backups of Cockpit sensor logs, to be retrieved in case of failure.',
   })
 
+  cockpitTemporaryLogsDB = localforage.createInstance({
+    driver: localforage.INDEXEDDB,
+    name: 'Cockpit - Temporary Sensor Log points',
+    storeName: 'cockpit-temporary-sensor-logs-db',
+    version: 1.0,
+    description: 'Temporary storage of Cockpit sensor log points.',
+  })
+
   /**
    * Start an intervaled logging
+   * @param {string} requesterId The ID of the requester. Can be any string, but should be unique. It will be used to
+   * stop the logging individually, so one log requester does not interfere with another.
    */
-  startLogging(): void {
-    if (this.logging()) {
-      console.warn('Tried to start logging but there was already a log being generated.')
+  startLogging(requesterId: string): void {
+    this.logRequesters.push(requesterId)
+    if (this.logRequesters.filter((id) => id !== requesterId).length > 0) {
```

Comment:
Is there a lazy way to evaluate this, and trigger on the first ID that isn't the requester's?

Definitely nitpicky since we don't expect `logRequesters` to be anywhere near large enough for this to be a performance concern, but it still seems worth following good practice if it's reasonable to do so 🤷

Reply:
I think this would be overkill here 😅

Reply:
Fair enough 🤷